CONTRIBUTING.md

```diff
@@ -32,34 +33,67 @@ As a Redwood user, you're already familiar with the codebase `yarn create redwoo
 
-**How do I choose which one to use?** If you've installed or upgraded a dependency, use the "local package registry emulation" workflow; otherwise, use "copy and watch".
+> **I've used `yarn rw` before, but what's `yarn rwt`?**
+>
+> All workflows use `redwood-tools` (`rwt`), Redwood's companion CLI development tool.
 
-> Both workflows use `redwood-tools` (`rwt`), Redwood's companion CLI development tool.
+All right, let's go through the entire contributor workflow, from Redwood Framework to Redwood App.
 
 ### Installing Dependencies
 
-Before you do anything, you should run `yarn install` in the root directory of your local-copy of the Redwood Framework to install the necessary dependencies.
+First, in your local copy of the Redwood Framework, make sure you've got the `main` branch's latest changes (via a `git pull`), then run `yarn install` in the root directory to install all the dependencies.
 
 ```terminal
 cd redwood
 yarn install
 ```
 
-You'll also want to upgrade your Redwood App to the canary or so, just so you can be sure you're testing your contribution with all the most recent changes:
+Then, in your Redwood App, which you've either just created using `yarn create redwood-app` or already had lying around, you'll want to upgrade to the canary&mdash;Redwood's bleeding edge&mdash;just so you can be sure you're testing your contribution with all the most recent changes:
 
 ```terminal
 cd redwood-app # wherever your redwood-app happens to be, whether it's one of our templates or your own
 yarn rw upgrade -t canary
 ```
 
+### yarn rwt link
+
+Ok, now that everything's up to date, in your Redwood App, run:
+
+```bash
+yarn rwt link [RW_PATH]
+```
+
+where [RW_PATH] is the path to your local copy of the Redwood Framework. For example,
+
+```
+yarn rwt link ~/redwood
+```
+
+> You can set an env var so that you don't have to specify this. See [Specifying a `RW_PATH`](#specifying-a-rw_path).
+
+You'll be prompted to add the redwood workspace to your Redwood App; go ahead and say yes.
+
+Then you'll start seeing a ton of output&mdash;`yarn rwt link` is building the Redwood Framework, watching it for changes, and copying all that over into the redwood workspace in your Redwood App. Your Redwood App isn't using the packages in `node_modules/@redwoodjs` anymore, but the packages in this workspace, with your local changes. You can even install packakges or upgrade dependencies&mdash;it's really that simple.
+
+When you're done, make sure to `unlink` your Redwood App:
+
+```
+yarn rwt unlink
+```
+
+This'll restore your Redwood App to it's original state, reverting it to the version of Redwood that it orignally had installed. Next time you want to contribute, just run `yarn rwt link` again!
```

Comment:
"This will"  flows better off my tongue, but I guess it's personal preference.

Reply:
I opt to over-contract because it's closer to speech but I'm not rigid about it or anything if you think we should back off from it here--can make that change locally and push it up.

Reply:
Your call. You're the native here 🙂   But I did just notice something that's most definitely wrong in the paragraph. It should be "its", not "it's" 😄